pkg/agent/endpoints/workload/handler_test.go

```diff
@@ -558,7 +558,9 @@ func (s *HandlerTestSuite) TestValidateJWTSVID() {
 	})
 	s.Require().NoError(err)
 
-	jwtSigner := jwtsvid.NewSigner(jwtsvid.SignerConfig{})
+	jwtSigner := jwtsvid.NewSigner(jwtsvid.SignerConfig{
+		Issuer: "issuer",
```

Comment:
Do we want a test case here where issuer is not set? Just a thought, not a blocker

Reply:
Sure, i can add one real quick.